packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -241,7 +241,7 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
             className={this._classNames.calloutMain}
             onDismiss={this.dismiss}
             onScroll={onScroll}
-            shouldRestoreFocus={true}
+            shouldRestoreFocus={!this.props.popupShouldNotRestoreFocus}
```

Comment:
Would be nice if both new props were `shouldRestoreFocus` as it would make working across multiple components feel familiar.

Reply:
Sounds good.